ali-plugin-main/src/test/java/com/hp/alm/ali/idea/services/ThemeFeatureServiceTest.java

```diff
@@ -40,7 +40,7 @@ public void preClean() throws Throwable {
 
     @Test
     public void testGetFeatures() {
-        handler.addRequest("GET", "/qcbin/rest/domains/domain/projects/project/requirements?fields=id,name,release-backlog-item.id,release-backlog-item.blocked,product-group-id&query={product-group-id[1000]; type-id[71]}&order-by={name[ASC]}&page-size=1000", 200)
+        handler.addRequest("GET", "/qcbin/rest/domains/domain/projects/project/requirements?fields=id,name,release-backlog-item.id,release-backlog-item.blocked,product-group-id&query={type-id[71]; product-group-id[1000]}&order-by={name[ASC]}&page-size=1000", 200)
```

Comment:
Query parameters are added to map in order shown in picture below. With LinkedHashMap should work properly.
![badorder](https://cloud.githubusercontent.com/assets/1066074/6881294/301f321e-d559-11e4-87c5-ba0ea94ffff2.png)